qa/SidechainTestFramework/sc_boostrap_info.py

```diff
@@ -77,13 +77,30 @@ def __init__(self, sc_creation_info, *sc_nodes_configuration):
         self.sc_creation_info = sc_creation_info
         self.sc_nodes_configuration = sc_nodes_configuration
 
+
+"""
+An account.
+The JSON representation is only for documentation.
+
+Account: {
+    "privateKey": "a private key",
+    "publicKey": "a public key"
+}
+"""
+class Account(object):
+
+    def __init__(self, privateKey, publicKey):
+        self.privateKey = privateKey
```

Comment:
should be `secret`, but not a `privateKey`. Below you are using publicKey and full secret data, not a privateKey part of it.

Reply:
Done